web/themes/contrib/civictheme/templates/form/container.html.twig

```diff
@@ -0,0 +1,11 @@
+{#
+/**
+ * @file
+ * CivicTheme theme implementation of a container used to wrap child elements.
+ *
+ * @see template_preprocess_container()
+ *
+ * @ingroup themeable
+ */
+#}
+{% include '@molecules/basic-content/basic-content.twig' %}
```

Comment:
@richardgaunt This breaks the admin menu when logged in 
![Screenshot 2024-10-25 at 8 57 52 AM](https://github.com/user-attachments/assets/0df854af-5b5e-4f45-aa69-3dab26bbdea1)